spec/rubocop/cop/style/method_name_spec.rb

```diff
@@ -49,6 +49,16 @@ def +(other)
       END
     end
 
+    it 'accepts unary operator definitions' do
+      expect_no_offenses(<<-END.strip_indent)
```

Comment:
Which editors?

Btw, we can probably add a rule in the style guide about proper delim naming or something.